docs/v1/P4Runtime-Spec.mdk

```diff
@@ -679,6 +679,25 @@ described as follows:
       controller is determined to be a slave for a given (`device_id`,
       `role_id`).
 
+gRPC enables the server to identify which client originated each message in the
+`StreamChannel` stream. For example, in synchronous mode, the gRPC libraries
```

Comment:
can we replace "gRPC libraries" with "C++ gRPC library" here to be more accurate (streams work differently in gRPC Python for example)., and maybe add a reference to `[@gRPCStreamC]`

Reply:
Done, with reference added.